### PR TITLE
APIGOV-26925 Update topology/environments to reflect stage, add missing content for view API Service details

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 2a5ec80 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-379.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by ColeSammonsAxway — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
I approve

File: content/en/docs/connect_manage_environ/view_environments.md [hunk 1]
```diff
@@ -99,13 +99,27 @@ The API Service details page includes:
...
...iance Score**: Compliance validation is an optional entitlement, API Compliance, which can be purchased with the Amplify API Management Platform product.  Compliance validation is the verification of your APIs in terms of how they meet the guidelines defined in a ruleset. Both an API Design and Security ruleset can be applied to all the APIs in a selected environment. The API is checked for compliance in terms of the API methods supported or the presence of a security policy. The grading results of the compliance are measured in the number of errors, warnings, info, or hints found. In the Services tab, two columns visualize the compliance grading of each API. The first one shows the grade from a Security perspective, the second is the Design compliance level. Hover over the grading letter to see the summary of the analysis that resulted in the current grading. A activity indicator is displayed in either grading column once the API is queued for Design or Security compliance validation. An N/A is displayed in either grading column if the API is not one of the supported API types. A red icon is displayed in the grading column if the API specification file is missing a required value.
```
Comment (lbadenhop) [inline content/en/docs/connect_manage_environ/view_environments.md:102]: The hover functionality is on the Services tab, not the Version tab. Move the associated content to line 60?

**Comment by lbadenhop** [comment]
@ColeSammonsAxway, see comment

**Pull request merged**